content/en/docs/reference/access-authn-authz/mutating-admission-policy.md

```diff
@@ -0,0 +1,54 @@
+---
+reviewers:
+- deads2k
+- sttts
+- cici37
+title: Mutating Admission Policy
+content_type: concept
```

Comment:
Ideally (will have to happen post release) we also add the metadata to place API reference links here. Have a look at the concept page for Deployment to get an idea of what I mean (both source code and the rendered page).

Nice to have for alpha, but **not** required until beta.